src/ controllers/trainee/routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import TraineeController from './Controller';
+import Controller from './Controller';
+
+const traineeRouter = Router();
+
+traineeRouter.route('/')
+.get(TraineeController.create)
+.post(TraineeController.list)
+.delete(TraineeController.delete)
+.put(TraineeController.Update);
```

Comment:
please check this, it is not done in right way. 
get isfor listing 
post is for creating 
put is for updating 
delete for deleting 
Please go through these concepts properly

traineeRouter
    .get('/', trainee.get) 
    .post('/', trainee.create)
    .put('/', trainee.update)
    .delete('/:id', trainee.delete);